manifests/z/Ziad/TCG/6.8/Ziad.TCG.installer.yaml

```diff
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=NVS1.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Ziad.TCG
+PackageVersion: "6.8"
+InstallerLocale: en-US
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /verysilent
+  SilentWithProgress: /silent
+UpgradeBehavior: uninstallPrevious
+Commands:
+- temp_cleaner_gui
+ReleaseDate: 2023-07-03
+Installers:
+- Architecture: x64
+  InstallerUrl: https://liquidtelecom.dl.sourceforge.net/project/temp-cleaner-gui/temp_cleaner_gui_v6.8-stable_win7sp1to11_amd64_mrx.exe
+  InstallerSha256: A4C1123196C2FE4337F4B9F89C6E8F47DDC12E0105FFE0CA7434322EFFCFF792
+- Architecture: x86
+  InstallerUrl: https://liquidtelecom.dl.sourceforge.net/project/temp-cleaner-gui/temp_cleaner_gui_v6.8-stable_win8to10_x86_mrx.exe
+  InstallerSha256: 2F3E2E18628F407C92CD67BC6816CB6DFA51306A103E76C46B0A71F076AAF758
```

Comment:
```suggestion
  InstallerSha256: 2F3E2E18628F407C92CD67BC6816CB6DFA51306A103E76C46B0A71F076AAF758
  AppsAndFeaturesEntries:
    - DisplayVersion: 6.8-stable
      ProductCode: '{3F0763AD-99A6-412F-A862-E1E379A6B4D2}_is1'
```

Reply:
I also didn't want to include the Product code as I thought it is device dependent.

Reply:
okay.
The interactive script `Yamlcreate.ps1` said it is **Optional** to include a ProductCode that's why I didn't include one.